### ISSUE TITLE
terra.tx.hash returns incorrect value

### ISSUE THREAD

**Issue by sergiosemenukha** [terra-money/terra.py#129]
- terra_sdk version: 2.0.6
- Python version: 3.8
- Operating System: Windows 10 pro

### Description

Getting an incorrect transaction hash value.
E.g. TXs list for block 7318543:
```
26f67ec1a78609e7a82fe3f7bc0014c2ffe4060368a602df05d66e143957a080
c0d8cdcc4cca588aa4261873b68dcbb51f7e6e655922902f87982fdcd3ffd6f7
645e9b23a9a4e31410b4f7f83e54b754462ba00e26e78a72fa1ce5cacfe74b00
3f930a46dcf27c67160594bbbaf74888a2c6592e71692725cb2040c9eff52e82
6654e82bfffe65add64dd0ddfcb12adc42a4ec0ca9aa8c600e79cf142335a67e
9571559aabb09cc8484c30b17e1ad7de692445c33f4189573d03722461b9a6ef
7fdaece3f6dca326d159e5c20e1abf46a2149a861e7035bc583b9e31a86c09b0
f89da10c5751d77c69a9fc395210a3ab7b7b5d59bb59f44934ae36faf048ad59
9acf4bda31c6d46fa0b67680fed737dd2efbebd591b65848c82174358950bf11
3fccecef45b393548cbab99453786d2c1bfb3273590a3a3547e2f13f869b7646
4a24143ede51ace5d36224d8be26a95d860f722a21540c0d5b481f210b8024f7
6b77e63be10f5de74eaa522b7fc8782d3b36bb0f84c5deec08b0774fdfb2c031
6dcd17a1fe9ac6a743db3ecb419352909876bc1188f9288ece579cf309f01e7a
8cc65d106b35a3928d3cca311f437cb7a10ed6f1fdbd5e0b46ef362427788f07
0dc5be5f32acc56a5ff738efc1a8d877e50de10cb2f100e53b7b28178db17916
141c08a9609448db70c4c1f87c48f584c2d388721e66cab73d18b54406c41f6d
30ad1445c889ef3e3d24e1ea98acb22056c2451db2db1c698d64c4fe9bce7b64
```

### What I Did

```
terra = LCDClient(chain_id=settings.terra_chain_id, url=settings.terra_url)
block = terra.tendermint.block_info(7318543)
transactions = []

for txraw in block["block"]["data"]["txs"]:
	tx = terra.tx.decode(txraw)
	tx_hash = terra.tx.hash(tx)
```

**Comment by Vritra4** [comment]
fixed with #127 by @Aureliano90. thanks @Aureliano90!

**Comment by Aureliano90** [comment]
> fixed with #127 by @Aureliano90. thanks @Aureliano90!

Actually, not quite. My test stopped at `30ad1445c889ef3e3d24e1ea98acb22056c2451db2db1c698d64c4fe9bce7b64` as well.

The error is
```
\terra_sdk\core\bank\msgs.py", line 225, in from_proto
    inputs=[MultiSendInput.from_proto(x) for x in proto["inputs"]],
TypeError: 'MsgMultiSend' object is not subscriptable
```

**Comment by Vritra4** [comment]
@Aureliano90 that is fixed with #130
